pkg/clusteragent/clusterchecks/api.go

```diff
@@ -88,3 +88,10 @@ func (h *Handler) GetAllEndpointsCheckConfigs() (types.ConfigResponse, error) {
 	}
 	return response, err
 }
+
+func (h *Handler) RebalanceClusterChecks() (types.ResultResponse, error) {
```

Comment:
we should update the stats before triggering the rebalance like in [here](https://github.com/DataDog/datadog-agent/blob/master/pkg/clusteragent/clusterchecks/dispatcher_main.go#L198-L204) 
```suggestion
func (h *Handler) RebalanceClusterChecks() (types.ResultResponse, error) {
    h.dispatcher.updateRunnersStats()
```

Also I think we should check if the feature (advanced dispatching) is enabled and return an error if it's not